hummingbot/client/ui/hummingbot_cli.py

```diff
@@ -74,6 +74,10 @@ def accept(self, buff):
         self.log(output)
         self.input_handler(self.input_field.text)
 
+    def reset(self):
+        self.pending_input = "exiting create command configuration..."
```

Comment:
I don't think this input is needed since the message in `keybindings.py` is already printed

<img width="613" alt="Screen Shot 2020-05-21 at 5 03 00 PM" src="https://user-images.githubusercontent.com/1795030/82617453-f2388480-9b84-11ea-80ff-ca6b4a300b92.png">

Reply:
CTRL-C is already in use for copy functionality.

Reply:
ok thanks - CTRL-X is fine.